sphinx/setup_command.py

```diff
@@ -105,7 +105,8 @@ def initialize_options(self):
         self.config_dir = None  # type: unicode
         self.link_index = False
         self.copyright = ''
-        self.verbosity = 0
+        # Link verbosity to distutils' (which uses 1 by default).
+        self.verbosity = self.distribution.verbose - 1
```

Comment:
mypy warns this line as following. Could you check this please?:
>sphinx/setup_command.py:109:26: error: "BuildDoc" has no attribute "distribution"
https://travis-ci.org/sphinx-doc/sphinx/jobs/391888857#L493